images/test.cci-export.Dockerfile

```diff
@@ -1,9 +1,8 @@
 ARG BASE_TAG
```

Comment:
Is this file still needed? Or just renamed?

Reply:
it is still called cci-export in dependent repos. so yeah. still needed.